inference-engine/thirdparty/clDNN/runtime/ocl/ocl_event.cpp

```diff
@@ -47,20 +47,24 @@ void base_event::set_ocl_callback() {
     }
 }
 
-void base_event::wait_impl() {
+void ocl_event::wait_impl() {
     if (_event.get() != nullptr) {
         _event.wait();
     }
 }
 
-bool base_event::is_set_impl() {
+void ocl_event::set_impl() {
+    wait_impl();
+}
+
+bool ocl_event::is_set_impl() {
     if (_event.get() != nullptr) {
         return _event.getInfo<CL_EVENT_COMMAND_EXECUTION_STATUS>() == CL_COMPLETE;
     }
     return true;
 }
 
-bool base_event::add_event_handler_impl(event_handler, void*) {
+bool ocl_event::add_event_handler_impl(event_handler, void*) {
```

Comment:
I found that `add_event_handler` is not used at all. Am I missing something?

Reply:
I tested some networks after removing the function, and it seems to be OK..

Reply:
Yes, it's not used. As far as I remember it was added long ago for debugging purposes. So we can remove it if we don't think that it may be useful, but I think it can be done in separate PR.